app/database/__init__.py

```diff
@@ -39,8 +40,37 @@ def get_engine(uri):
     return create_engine(uri, **options)
 
 
+def get_async_engine(uri):
+    options = {
+        "pool_recycle": 3600,
+        "pool_size": 10,
+        "pool_timeout": 30,
+        "pool_pre_ping": True,
+        "max_overflow": 30,
+        "echo": config.DB_ECHO,
+    }
+    return create_async_engine(uri, **options)
+
+
+# Create Engine
 engine = get_engine(config.DATABASE_URL)
+if engine.dialect.name == "mysql":
+    MYSQL_ASYNC_DATABASE_URL = config.DATABASE_URL.replace(
+        "mysql+pymysql://", "mysql+aiomysql://"
+    )
+    async_engine = get_async_engine(MYSQL_ASYNC_DATABASE_URL)
+else:
+    async_engine = get_async_engine(config.DATABASE_URL)
```

Comment:
When using MySQL as the DB backend, the URI string is converted for use with `aiomysql`.